VideoRenderer/VideoRenderer/VideoStreamViewController.swift

```diff
@@ -74,7 +74,7 @@ public final class VideoStreamViewController: UIViewController, RendererProtocol
         provider: { VideoStreamViewController() }
     )
     
-    private var observer: PlayerObserver?
+    private var observer: SystemPlayerObserver?
     private var pictureInPictureObserver: PictureInPictureControllerObserver?
     
     private var timeObserver: Any?
```

Comment:
How about move `timeObserver` into `SystemPlayerObserver`? It has similar life cycle and used in same places where we use `SystemPlayerObserver`.